### PR TITLE
minimal leroi run script

### PR DESCRIPTION
**Pull request by jedman**
minimal script to run Leroi without lots of local file manipulation. all outputs end up in a specified directory and are named `Leroi.*`

compare this script to `Leroi/Leroi`...

### CODE REVIEW

File: Leroi/leroi-kobold [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+# leroi-kobold $1 $2
+# first arg is name of the .cfl file to run, second arg is the directory to run and store the outputs
```
Comment (jen-holt-kobold) [inline Leroi/leroi-kobold:3]: This assumes either $1 is in $2 or has it's full path included. Might be worth adding that to the comment

File: Leroi/leroi-kobold [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+# output file names are "Leroi*"
+
+# move to job directory
+test -d "$2" && cd "$2"
+
+# copy file to Leroi.cfl
+test -f "$1" && cp "$1" Leroi.cfl
```
Comment (jen-holt-kobold) [inline Leroi/leroi-kobold:10]: We should check $2 for write permissions either here or from the python wrapper that calls this

**Review by jen-holt-kobold — APPROVED** [review]
Small comments, nothing major

**Comment by devjit-kobold** [comment]
Why are we doing this in bash? could we put the files in a directory and call Leroi with that working directory? This is probably best handled by python / the client...

I guess the trick is that the file must be named 'Leroi.cfl' - so maybe symlink a file in the directory we want the outputs? I would avoid additional layers of redirection though - so if we can just write Leroi.cfl directly - that's preferable...

**Comment by jedman** [comment]
I think @devjit-kobold is right, let's just handle this in the python wrapper

**Pull request closed without merging**